docs/core/whats-new/dotnet-10/runtime.md

```diff
@@ -83,6 +83,29 @@ static void Sum()
 
 Because the JIT knows `numbers` is an array of only three integers at compile time, and it doesn't outlive a call to `Sum`, it allocates it on the stack.
 
+## Stack allocation of small arrays of reference types
+
+Building on the stack allocation improvements introduced in .NET 9, .NET 10 Preview 3 extends this optimization to small arrays of reference types. Previously, arrays of reference types were always allocated on the heap, even when their lifetime was scoped to a single method. Now, the JIT can stack-allocate such arrays when it determines that they don't outlive their creation context. For example:
```

Comment:
```suggestion
Building on the stack allocation improvements introduced in .NET 9, .NET 10 extends this optimization to small arrays of reference types. Previously, arrays of reference types were always allocated on the heap, even when their lifetime was scoped to a single method. Now, the JIT can stack-allocate such arrays when it determines that they don't outlive their creation context. For example:
```

Reply:
This article should be .NET 10 all-up, without regard to which preview introduced the change. At least for .NET 9, we decided that was what the blog posts were for (point in time).

Reply:
Agreed, that was a copy/paste mistake I just missed